examples/tools/scene_viewer/main.rs

```diff
@@ -0,0 +1,179 @@
+//! A simple glTF scene viewer made with Bevy.
+//!
+//! Just run `cargo run --release --example scene_viewer /path/to/model.gltf`,
+//! replacing the path as appropriate.
+//! In case of multiple scenes, you can select which to display by adapting the file path: `/path/to/model.gltf#Scene1`.
+//! With no arguments it will load the `FlightHelmet` glTF model from the repository assets subdirectory.
+
+use bevy::{
+    math::Vec3A,
+    prelude::*,
+    render::primitives::{Aabb, Sphere},
+};
+
+mod camera_controller_plugin;
+mod scene_viewer_plugin;
+
+use camera_controller_plugin::{CameraController, CameraControllerPlugin};
+use scene_viewer_plugin::{SceneHandle, SceneViewerPlugin};
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+struct CameraControllerCheckSystem;
```

Comment:
We can probably eliminate this type: typed system labels rarely make sense when they're non-pub.